qiita_pet/support_files/doc/source/checklist-for-ebi-ena-submission.rst

```diff
@@ -0,0 +1,79 @@
+Checklist to send data to EBI-ENA
+=================================
+
+For each preparation that needs to be uploaded to EBI-ENA we will check:
+
+  1. Data processing
+
+    a. Make sure there is raw data linked - no BIOMs allowed
+    b. The data is processed and the owner confirms the data is correct:
+
+      1. For target gene: data is demultiplexed (review split_library_log to make sure it seems fine) and there is at least a close reference (GG for 16S, Silva for 18S, UNITE for ITS) or trim/deblur artifacts. Trimming should be done with 90, 100 and 150 base pairs (prefered).
+      2. For shotgun: data is uploaded via per_sample_FASTQ and processed using Shogun/utree
+
+  2. Verify the sample information
+
+    a. Check that the sample information file complies with `the current Qiita metadata format <https://qiita.ucsd.edu/static/doc/html/gettingstartedguide/index.html#sample-information-file>`__
+    b. Minimal information:
+
+      1. *sample_name*
+      2. *host_subject_id*
+      3. *sample_type*
+      4. *taxon_id* - needs to match *scientific_name* value
+      5. *scientific_name* - needs to match *taxon_id* value - this is the name of the `metagenome <https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Tree&id=12908&lvl=3&srchmode=1&keep=1&unlock>`__ referenced in the column *taxon_id* and that the two values match.  Submission will not work if the user puts *host_scientific_name* or *host_taxid* instead.  Do not accept EBI null values. For null values use *scientific_name* “metagenome” and *taxon_id* “256318”
+      6. *env_biome*, *env_feature*, *env_material*, *env_package*
```

Comment:
Can you add a link like in the point above where the possible values are listed?